packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -26,6 +26,8 @@ const COLORS: Partial<ExportedComponentLevelTokens>[] = [
   "countryFlag",
 ];
 
+const px = (n: number) => `${n}px`;
```

Comment:
I believe these changes should be in a different commit with scope `tailwind`

Reply:
do not think so, it's still will be a chore that later will be squashed